tools/run_pr_builder.sh

```diff
@@ -18,7 +18,17 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
 go build ${GO_LIST}
-go test ${GO_LIST}
+
+# go test plus build coverage data for upload codecov.io
+echo "" > coverage.txt
+for d in ${GO_LIST}; do
+    go test -coverprofile=profile.out -covermode=atomic $d
+    if [ -f profile.out ]; then
+        cat profile.out >> coverage.txt
+        rm profile.out
+    fi
+done
+bash <(curl -s https://codecov.io/bash) -t ${COVERALLS_TOKEN}
```

Comment:
I will update Jenkins to have this environment variable set appropriately in order to avoid having the credentials in this script.